qc/qc_adtte.R

```diff
@@ -0,0 +1,13 @@
+###########################################################################
+#' developers : Robert Devine/
```

Comment:
the '/' can be removed

Reply:
Thanks @laxamanaj and @SHAESEN2.

1. > styler::style_file("qc/qc_adtte.R")
Styling  1  files:
 qc/qc_adtte.R ℹ 
────────────────────────────────────────
Status	Count	Legend 
✔ 	0	File unchanged.
ℹ 	1	File changed.
✖ 	0	Styling threw an error.
────────────────────────────────────────
Please review the changes carefully!

2. '/' removed